src/assets/icon/SearchArrow.jsx

```diff
@@ -0,0 +1,9 @@
+const SearchArrow = () => {
+    return (
+<svg width="24" height="24">
```

Comment:
Do not use magic numbers like 24
Pass it through the props

Reply:
fixed